client/additional-methods-setup/upe-preview-methods-selector/enable-upe-preview-task.scss

```diff
@@ -0,0 +1,77 @@
+.enable-upe-preview {
+	&__body {
+		&.is-active {
+			// there's so much text that on mobile the text overflows the max-height set on the collapsible body.
```

Comment:
Without this on mobile:
![Screen Shot 2021-07-14 at 10 47 30 AM](https://user-images.githubusercontent.com/273592/125652133-aef5bf8c-54a6-4d36-aeb0-f240fd862158.png)